meter.go

```diff
@@ -208,11 +229,12 @@ func (m *StandardMeter) tick() {
 type meterArbiter struct {
 	sync.RWMutex
 	started bool
-	meters  []*StandardMeter
+	meters  map[int64]*StandardMeter
```

Comment:
What about using a set of `*StandardMeter` (aka `map[*StandardMeter]struct{}`) to avoid id collision/overflow yet benefits from fast and easy deletion?

Reply:
Yes, I think that would be a better solution than adding the ID generation